src/client/components/InputText.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import TextField from "@material-ui/core/TextField";
+class InputText extends Component {
```

Comment:
Please make this a functional stateless component, rename it to InputEmail and rename the file to InputEmail.js